.github/workflows/quick-manual-model-test.yml

```
@@ -47,3 +47,9 @@ jobs:
           ersilia sample -n 5 -f input.csv
           ersilia -v api -i input.csv
           ersilia close
+
+      - name: Upload log output
```

```suggestion
      - name: Upload log output
        if: ${{ failure() }}
```

Since we only want to upload an artifact with error logs, should we gate this step to only run when something fails?